client/app/(auth)/hooks/api/index.ts

```diff
@@ -1,3 +1,4 @@
-export { useLoginMutation } from './useLoginMutation'
+export { useAccountMutation } from './useAccountMutation'
```

Comment:
This one too, move to `app/account/hooks/api` ?

The feature directory rule is that you can not import from another feature!

Reply:
fixed it.
52ad6740cc927e9c3e1817baca316720d513360f